pkg/operation/common/utils.go

```diff
@@ -431,7 +431,7 @@ func DeleteLoggingStack(k8sClient kubernetes.Interface, namespace string) error
 
 	var (
 		services     = []string{"kibana-logging", "elasticsearch-logging", "fluentd-es"}
-		configmaps   = []string{"kibana-index-registration", "curator-hourly-config", "curator-daily-config", "fluent-bit-config", "fluentd-es-config"}
+		configmaps   = []string{"kibana-object-registration", "kibana-saved-objects-config", "curator-hourly-config", "curator-daily-config", "fluent-bit-config", "fluentd-es-config"}
```

Comment:
The ConfigMap is not called `kibana-saved-objects-config` but `kibana-saved-objects`